modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -160,5 +160,25 @@ def extract_poa_code(poa_form_data)
         poa_form_data['representative']['poaCode']
       end
     end
+
+    def log_service_progress(claim_id, tag, detail)
+      log_tag = tag == 'pdf' ? '526_v2_PDF_Generator_job' : '526_v2_Docker_Container_job'
```

Comment:
@mchristiansonVA  Wonder if it is worth it to refactor this since we are here.  Right now it uses `526_v2_PDF_Generator_job` and `526_v2_Docker_Container_job` just like the async jobs.  My question is do we think it would be valuable to refactor this by:
* adding a tag in the service files similar to the sidekiq jobs
*  use `_service` in place of `_job` 
* then instead of this line we could just do `self.class::LOG_TAG` like on line 151 above here

Maybe it isn't worth the squeeze, but wanted to at least put the idea out there to see since it is already getting refactored.

Reply:
In fact looking at it we could probably do away with this whole entire method if we made those changes and jsut call the other logger, I _believe_